crates/bevy_pbr/src/render/pbr.wgsl

```diff
@@ -32,420 +32,13 @@
 //
 // The above integration needs to be approximated.
```

Comment:
Should this comment (and everything above it) go into `pbr_functions.wgsl` now?

Reply:
Yes. I did that on a branch that I had rebased on top of the hot core shader reloading but I've since discarded that branch as Cart came with a better suggestion for the reloading. I'll double-check whether there is anything else on that branch.

Reply:
Done.